### PR TITLE
Decouple contract

### PR DESCRIPTION
**Pull request by saipramod**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -77,41 +81,70 @@ app.get("/map", (req, res) => {
...
-    if (stateCode != null) {
-      dataToProcess = dataToProcess.filter((row) =>
-        row.county.endsWith(stateCode)
-      );
-    }
+app.get("/timeseries-data", (req, res) => {
+  const contacts = Object.keys(db);
+  const stateCode = req.query.stateCode;
+  const type: string = req.query.type;
+  const percentiles = ["2.5", "25", "50", "75", "97.5"];
```
Comment (lzl124631x) [inline src/app.ts:88]: I think we can extract these from the DB instead of hardcode them here.

File: src/app.ts [hunk 2]
```diff
@@ -3,11 +3,15 @@ import csv from "csv-parse";
 import fs from "fs";
 import cors from "cors";
 
-import { mapOwn, toSeries, toRangeTimeSeriesData } from "./util";
+import { mapOwn, toSeries } from "./util";
 import { StackedChartData } from "./payloads/stackedchart";
 import { Entry, MapDataEntry, MapData } from "./type";
 import { PERCENTILE_GROUPS } from "./chart-configuration/stackedchart-configuration";
```
Comment (lzl124631x) [inline src/app.ts:9]: May we remove the unused code?

File: src/app.ts (hunk 1 above)
Comment (lzl124631x) [reply src/app.ts:88]: If it's difficult to change that you can ignore this comment. I guess it's unlikely for them to add more percentiles so hardcoding in the backend should be fine. But let's make sure the front-end doesn't need hardcoding.

Comment (saipramod) [reply src/app.ts:88]: yup ! if it turns out they add more, i will remove the hardcoding.

**Pull request merged**